test_server.py

```diff
@@ -1,12 +1,18 @@
 import enet
-
 SHUTDOWN_MSG = "SHUTDOWN"
 
 host = enet.Host(enet.Address(b"localhost", 54301), 10, 0, 0, 0)
+def receive_callback(address, data):
+    global host
+    if data and data == "\xff\xff\xff\xffgetstatus\x00":
```

Comment:
Bytestring literal is what we are using in the game server, i.e. i'm copy pasting functionality from there so that tests are closer to production